automatic/vlc.portable/vlc.portable.nuspec

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>vlc.portable</id>
+    <title>VLC media player (Portable)</title>
+    <version>3.0.16</version>
+    <authors>VideoLAN Organization</authors>
+    <owners>chocolatey-community, Rob Reynolds</owners>
+    <summary>VLC Media Player</summary>
+    <licenseUrl>http://www.videolan.org/legal.html</licenseUrl>
+    <description><![CDATA[
+VLC is a free and open source cross-platform multimedia player and framework that plays most multimedia files as well as DVD, Audio CD, VCD, and various streaming protocols.
+
+![Screenshot](https://i.imgur.com/DEG19Fs.png)
+
+## Features
+
+- Plays Files, Discs, Webcams, Devices and Streams.
+- Fast - Hardware Decoding.
+- Plays most codecs with no codec packs needed - MPEG-2, MPEG-4, H.264, MKV, WebM, WMV, MP3...
+- Complete feature-set over the video, subtitle synchronisation, video and audio filters.
+- Runs on all platforms - Windows, Linux, Mac OS X, Unix, iOS, Android ...
+- Completely Free - no spyware, no ads and no user tracking.
+
+## Package parameters
+
+- `/Language:en` - Two letter language code: nl, en, fr ...
+
+
+Example: `choco install --params "/Language:fr"` *Will set the language of VLC to french*
+
+## Notes
+
+- Related packages: [vlc-skins](https://chocolatey.org/packages/vlc-skins) [vlc-nightly](https://chocolatey.org/packages/vlc-nightly)
+
+]]></description>
+    <projectUrl>http://www.videolan.org/vlc/</projectUrl>
+    <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-coreteampackages/tree/master/automatic/vlc</packageSourceUrl>
+    <projectSourceUrl>http://www.videolan.org/vlc/download-sources.html</projectSourceUrl>
+    <bugTrackerUrl>https://code.videolan.org/videolan/vlc/-/issues/</bugTrackerUrl>
+    <docsUrl>https://wiki.videolan.org/Documentation:Documentation/</docsUrl>
+    <mailingListUrl>https://forum.videolan.org/</mailingListUrl>
+    <tags>vlc foss cross-platform multimedia audio video mp3 dvd avi media player admin</tags>
+    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@3952380efd0776bea6d964ed4e28efa979b434d2/icons/vlc.png</iconUrl>
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.3.3" />
```

Comment:
It doesn't need it.

Reply:
Actually, it is needed for `Remove-Process`.